_posts/2020-09-08-secure-firmware-updates-with-code-signing.md

```diff
@@ -118,9 +118,8 @@ integers, each 32 bytes long.
 ## Firmware Signing Implementation
 
 Our implementation builds upon the code we wrote for our [firmware update
-architecture post](({% post_url 2020-06-23-device-firmware-update-cookbook %})).
-You may find that code on Github at
-https://github.com/memfault/interrupt/tree/20ec4ba2d0def6214aa808717446cdbaced1c352/example/fwup-architecture.
+architecture post]({% post_url 2020-06-23-device-firmware-update-cookbook%} ).
+You may find that code on Github at [interrupt@96186a0](https://github.com/memfault/interrupt/tree/20ec4ba2d0def6214aa808717446cdbaced1c352/example/fwup-architecture).
```

Comment:
I would generate a cta link with a shorter name, this is a suggestion, like the repo name  at commit hash?